packages/scrum-client/src/components/TeamList/TeamList.js

```diff
@@ -1,21 +1,14 @@
 import React from 'react';
 import { Badge, Row, Col, Figure } from 'react-bootstrap';
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, LabelList, Label } from 'recharts';
 import PropTypes from 'prop-types';
 import './TeamList.css';
 
 const getUserVote = (storyVotes, user) => {
-  // eslint-disable-next-line
-  console.log('storyVotes', storyVotes);
-  // eslint-disable-next-line
-  console.log('user._id', user._id);
-  // Object.keys(storyVotes).length === 0
-  // JSON.stringify(obj) === '{}';
-  // eslint-disable-next-line
-  console.log('Object.keys(storyVotes).length', Object.keys(storyVotes).length);
-  // eslint-disable-next-line
-  const storyVote = JSON.stringify(storyVotes) === '{}' ? undefined : storyVotes.find(([id]) => {
-          // eslint-disable-next-line
-    console.log('id', id);
+  const storyVote =
+    JSON.stringify(storyVotes) === '{}'
```

Comment:
JSON.stringify does convert to a string , not and object .. right ?
Is that comparison correct ?